### ISSUE TITLE
Changing size doesn't affect li elements

### ISSUE THREAD

**Issue by bricewge** [coldnew/org-ioslide#32]
The `li` elements aren't affected when changing the size of the text. The following patch should fix this. I didn't recompiled the theme as I don't have any ruby installed on my system.

``` diff
 theme/scss/default.scss | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/theme/scss/default.scss b/theme/scss/default.scss
index 9316d61..95afc2d 100644
--- a/theme/scss/default.scss
+++ b/theme/scss/default.scss
@@ -534,7 +534,7 @@ slide.fill {
 /* Size variants */
 
 article.smaller {
-    p, ul {
+    p, ul, li {
 	font-size: 20px;
 	line-height: 24px;
 	letter-spacing: 0;
@@ -560,7 +560,7 @@ article.smaller {
 }
 
 article.larger {
-    p, ul {
+    p, ul, li {
 	font-size: 36px;
 	line-height: 42px;
 	letter-spacing: 0;
@@ -586,7 +586,7 @@ article.larger {
 }
 
 article.large {
-    p, ul {
+    p, ul, li {
 	font-size: 42px;
 	line-height: 52px;
 	letter-spacing: 0;
```

**Comment by coldnew** [comment]
Fixed in #33 thanks :)

**Comment by bricewge** [comment]
That was quick. Thanks!
